src/mumble/Settings.h

```diff
@@ -345,6 +345,7 @@ struct Settings {
 	bool bReconnect;
 	bool bAutoConnect;
 	bool bQoS;
+	bool bDisablePublicList;
```

Comment:
```suggestion
	/// Disables the "Public Internet" section in the connect dialog if set.
	bool bDisablePublicList;
```
Why did you remove the comment in the first place?

Reply:
Tbh, i dont even know anymore that i removed that...